include/nitro/broken_options/argument.hpp

```diff
@@ -99,12 +100,27 @@ namespace broken_options
             return is_value() || static_cast<bool>(value_);
         }
 
+        bool has_prefix() const
+        {
+            return nitro::lang::starts_with(name_,"--no-");
+        }
+
     public:
         const std::string& data() const noexcept
         {
             return arg_;
         }
 
+        std::string name_without_prefix() const
+        {
+            if (!has_prefix())
+            {
+                raise<parser_error>("Trying to get the name without prefix but prefix is not given.");
+            }
+            std::string tmp = name();
+            return tmp.erase(0,5);
```

Comment:
Instead of the explicit copy and erase of the first few characters, you can use the `substr` member function of `std::string` directly on `name()`